core/src/test/scripts/system_tests_validate_data.py

```diff
@@ -93,7 +93,7 @@ def test_exit_status_success(self):
         # execute main function with arguments provided as if from sys.argv
         args = validateData.interface(args)
         exit_status = validateData.main_validate(args)
-        self.assertEqual(0, exit_status)
+        self.assertEqual(3, exit_status)
```

Comment:
Are we expecting to see warning when we validate `study_es_0`? I think `study_es_0` should pass the validation without any warnings and errors.

Reply:
Reverted to 0.